### PR TITLE
Jitr demo3

### PR DESCRIPTION
**Pull request by yehtarnsu**
Fixes #

### CODE REVIEW

File: lambda-assets/device-activator/app.ts [hunk 1]
```diff
@@ -162,4 +168,10 @@ export const handler = async (event: any = {}) : Promise <any> => {
     verifierName: verifierName,
   });
   return message;
-};
\ No newline at end of file
+};
+
+function getCommonName(certificatePem: string) {
+  const certificate = pki.certificateFromPem(certificatePem);
+  const commonName = certificate.subject.getField('CN').value;
```
Comment (MinCheTsai) [inline lambda-assets/device-activator/app.ts:175]: if common name not defined?

File: lambda-assets/ca-registrator/app.ts [hunk 2]
```diff
@@ -117,21 +117,63 @@ export const handler = async (event: any = {}) : Promise <any> => {
         throw new InformationNotFoundError(error.message);
       });
 
+    await s3Client.send(
+      new PutObjectCommand({
+        Bucket: bucketName,
+        Key: path.join(bucketPrefix || '', certificateId!, 'ca.public_key.pem'),
```
Comment (MinCheTsai) [inline lambda-assets/ca-registrator/app.ts:123]: use || or ??

File: lambda-assets/device-activator/app.ts (hunk 1 above)
Comment (yehtarnsu) [reply lambda-assets/device-activator/app.ts:175]: Using device certificate ID.

File: lambda-assets/ca-registrator/app.ts (hunk 2 above)
Comment (yehtarnsu) [reply lambda-assets/ca-registrator/app.ts]: Bucket prefix is defined previously. Remove this OR operator.

**Review by MinCheTsai — APPROVED** [review]

**Comment by MinCheTsai** [comment]
Changes approved, need resolve conflicts and merge.

**Pull request merged**